pkgs/ok_http/jnigen.yaml

```diff
@@ -0,0 +1,80 @@
+# To regenerate the JNI Bindings, download the OkHttp 4.12.0 JAR file from the Maven Repository
+# and place them in 'jar/'.
+# https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp/4.12.0
```

Comment:
It would be good to use that approach, but I used the one in `java_http` as this package is also expected to support other platforms down the line.
I guess it would be clearer if the JAR files were manually added to the project (non-Android).

I'll make an issue for a fix to this and/or further discussion ✅